src/documentation/pages/components/divider/examples/index.vue

```diff
@@ -0,0 +1,34 @@
+<template lang="pug">
+  div
+    h2 Examples
+    p.-text To render a divider, apply the class <code>chi-divider</code> to an <code>hr</code> or <code>div</code> tag.
+
+    div
+      <Base />
+      <Vertical />
+      <Inverse />
+      <Label />
+
+    div(v-if="['lumen', 'centurylink'].includes($store.state.themes.theme)")
+      <AdditionalSizesLumenCenturyLink />
```

Comment:
@evansito can you remove Addtional? I think sizes is more consistent with the rest of our doc